### PR TITLE
Fix #1545 - Parse null json

### LINKED ISSUE #1545: NPE thrown when null JSON passed to parseJson helper
We should either return an empty object or report the error.

### PR DESCRIPTION
**Pull request by TuxedoFish**
fixes #1545 

- Add 2 unit tests for if null is passed in (TAGTYPE.Section as an extra condition)
- Add 2 unit tests for if "{}" is passed in (TAGTYPE.Section as an extra condition)  

- Fix these two edge cases by adding a conditional at the beginning and returning an empty map in this case

### CODE REVIEW

File: src/main/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/ParseJsonHelper.java [hunk 1]
```diff
@@ -45,9 +50,14 @@ public Object apply(Object context, Options options) throws IOException {
             variableName = options.params.length > 0 ? options.param(0) : null;
         }
 
-        Object result = null;
+        Object result = emptyJson;
```
[the remark this answers is not in the document]
Comment (TuxedoFish) [reply src/main/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/ParseJsonHelper.java:53]: Yes, that is neater thanks I'll make that change

**Comment by Funtic** [comment]
Why is the result (emptyJson) immediately returned without being stored in a variable, if any?
